src/main/java/PeselValidator.java

```diff
@@ -0,0 +1,134 @@
+
+public class PeselValidator {
+
+
+    public boolean validate(String pesel) {
+
+        if (!checkPeselLength(pesel)) {
+            return false;
+        }
+
+        if (!checkPeselType(pesel)) {
+            return false;
+        }
+
+        if (!checkPeselMonth(pesel)) {
+            return false;
+        }
+
+        if (!checkPeselDay(pesel)) {
+            return false;
+        }
+
+        if (!checkPeselLastNumber(pesel)) {
+            return false;
+        }
+
+
+        return true;
+    }
+
+
+    private boolean checkPeselLength(String pesel) {
+
+        if (pesel.length() != 11) {
+            throw new NumberFormatException();
+
+        } else {
+            return true;
+        }
+    }
+
+    private boolean checkPeselType(String pesel) {
```

Comment:
I don't understand what this function validates - it will always return true and does not throw exception because exception is handled in try-catch block. I think you can just remove it.